synapse/module_api/__init__.py

```diff
@@ -700,6 +721,53 @@ def read_templates(
             (td for td in (self.custom_template_dir, custom_template_directory) if td),
         )
 
+    def is_mine(self, id: Union[str, DomainSpecificString]) -> bool:
+        """
+        Checks whether an ID comes from this homeserver.
+
+        Added in Synapse v1.44.0.
```

Comment:
Could you add `Args` and `Returns` sections to the docstring of this function and the one below please?

I was initially confused that this method wasn't just for user IDs :slightly_smiling_face: